src/controllers/signAccountOp/signAccountOp.ts

```diff
@@ -287,6 +288,8 @@ export class SignAccountOpController extends EventEmitter {
 
     if (estimation) this.#estimation = estimation
 
+    if (accountOp) this.accountOp = accountOp
```

Comment:
was this discussed with Dancho? makes sense but do we have internal mutability of accountOp in signAccountOp? we probably do when setting fees and etc. 

so this sounds like a potential bug